src/Minter.t.sol

```diff
@@ -42,6 +58,21 @@ contract MinterTest is Test, ERC721Holder {
     );
   }
 
+  function testUnauthorizedRangeMint() public {
+    balot.transferOwnership(address(minter));
+
+    address collection = address(balot);
+    uint16 start = 1;
+    uint16 end = 300;
+
+    vm.expectRevert();
```

Comment:
```suggestion
    vm.expectRevert(bytes("Ownable: caller is not the owner"));
```

Source: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/b53c43242fc9c0e435b66178c3847c4a1b417cc1/contracts/access/Ownable.sol#L43

Reply:
For some reason this change doesn't work as expected, and the test fails.

Only the following works: `vm.expectRevert(bytes(""));`

As I believe this is not critical and the previous code takes less assumptions, I will revert to the previous code with `vm.expectRevert()`

Reply:
But then if it reverts without reason, are we absolutely sure it reverts because "the caller is not the owner"? Yes, right because you tested it in JS too.

Reply:
Indeed, the `Minter.safeMintRange by attacker` test case in JS tries the same kind of call: `Minter.safeMintRange` from a non-owner signer. There the revert message is explicit.